Gemfile

```diff
@@ -6,3 +6,7 @@ gem 'rake'
 gem 'rspec'
 gem 'govuk-dummy_content_store', '0.0.5'
 gem 'foreman', '0.78.0'
+
+group :test do
+  gem 'pry-byebug'
```

Comment:
Maybe cherry-pick this out into its own PR as isn't related directly to this group of changes?